sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/percentiles.scala

```diff
@@ -324,3 +337,117 @@ case class Percentile(
     frequencyExpression = newThird
   )
 }
+
+/**
+ * The Percentile aggregate function returns the percentile(s) based on a discrete distribution of
+ * numeric column `expr` at the given percentage(s) with value range in [0.0, 1.0].
+ *
+ * Because the number of elements and their partial order cannot be determined in advance.
+ * Therefore we have to store all the elements in memory, and so notice that too many elements can
+ * cause GC paused and eventually OutOfMemory Errors.
+ */
```

Comment:
We need to write SQL function doc for it.

Reply:
OK

Reply:
This is not addessed yet.

Reply:
This is SQL fun doc: https://github.com/apache/spark/pull/35041/files#diff-b18693ab805b2074e50e41ac0e04bbc7a8cb6a3cc944419107bf383bbbcaa82fR243

Reply:
Ah, this is not a common SQL function, it has a special SQL syntax. nvm

Reply:
Yes. we only use this function in special syntax.